sdk/compute/Azure.ResourceManager.Compute/tests/Helpers/ResourceDataHelper.cs

```diff
@@ -339,5 +393,44 @@ public static VirtualMachineScaleSetData GetBasicLinuxVirtualMachineScaleSetData
             };
         }
         #endregion
+
+        #region ProximityPlacementGroup
+        public static void AssertProximityPlacementGroup(ProximityPlacementGroupData group1, ProximityPlacementGroupData group2)
+        {
+            Assert.AreEqual(group1.Name, group2.Name);
+            Assert.AreEqual(group1.Id, group2.Id);
+            Assert.AreEqual(group1.Location, group2.Location);
+            Assert.AreEqual(group1.Tags, group2.Tags);
```

Comment:
Actually we can use the function `AssertTrackedResource` to validate `Id, Name, Location, Tags and Type` are the same between these two instances, and add some more assertion clauses to validate other data in this model.

Reply:
Same as above